prow/spyglass/lenses/dumplenses.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lenses
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+)
+
+type FakeArtifact struct {
```

Comment:
Please put this back into a test file. This should not ship with the code.

Reply:
The problem is it can't then be used from the `junit` subfolder.
The solution is https://stackoverflow.com/a/38715325

Reply:
Copy the code in both locations or put it in its own testing package.

This doesn't belong in prod

Reply:
sg, I'll copy the code

Reply:
done